server/raft.go

```diff
@@ -1269,8 +1270,18 @@ func (n *raft) isCurrent(includeForwardProgress bool) bool {
 		return false
 	}
 
+	// If we were previously logging about falling behind, also log when the problem
+	// was cleared.
+	clearBehindState := func() {
+		if n.hcbehind {
+			n.warn("Health check OK, no longer falling behind")
```

Comment:
can we add details here about what asset it is? also on the falling behind log

Reply:
The `n.warn` is already prefixing with the Raft ID & group, I don't think `*raft` actually knows the real name of the asset though.

Reply:
ah of course, good (well unfortunately always logging the raft group names not the asset but ok its something)